fab/core.py

```diff
@@ -48,10 +48,9 @@ def __init__(self,
                              "flow_alpha_2_div_unbiased", "flow_alpha_2_div_nis",
                              "target_forward_kl"]
         if loss_type in EXPERIMENTAL_LOSSES:
-            warnings.warn("Running using experiment loss not used within the main FAB paper.")
+            raise Exception("Running using experiment loss not used within the main FAB paper.")
```

Comment:
Changed this warning to instead be an exception, as users shouldn't need these options, and I haven't used them in a while so they may have issues given the changes to the codebase.